_monitoring-plugins/pa/index.md

```diff
@@ -1,73 +1,110 @@
 ---
 layout: default
-title: Performance Analyzer
+title: Performance analyzer new
+parent: pa
 nav_order: 58
 has_children: true
 redirect_from:
   - /monitoring-plugins/pa/
 ---
 
-# Performance Analyzer
+# Performance analyzer
 
-Performance Analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics, independent of the Java Virtual Machine (JVM). PerfTop is the default command line interface (CLI) for displaying those metrics.
+Performance analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
 
-To download PerfTop, see [Download](https://github.com/opensearch-project/perftop/releases) on the PerfTop release page.
+The performance analyzer plugin is installed by default in OpenSearch version 2.0 and higher.
+{: .note }
 
-You can also install it using [npm](https://www.npmjs.com/):
+## Performance analyzer installation and configuration
 
-```bash
-npm install -g @aws/opensearch-perftop
-```
+Below are the steps to install and configure the performance analyzer plugin.
 
-![PerfTop screenshot]({{site.url}}{{site.baseurl}}/images/perftop.png)
+### Install
 
-For enabling Performance Analyzer with tarball installations of OpenSearch, see [Configure Performance Analyzer for Tarball Installation](#configure-performance-analyzer-for-tarball-installations).
+The performance analyzer plugin is included in both the [Docker]({{site.url}}{{site.baseurl}}/opensearch/install/docker/) and [tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/) installs of OpenSearch. In the event that you need to install the performance analyzer plugin manually, download the plugin from [Maven](https://search.maven.org/search?q=org.opensearch.plugin). Then install the plugin using the standard [plugins install]({{site.url}}{{site.baseurl}}/opensearch/install/plugins/) process. Performance analyzer will run on each node in a cluster.
 
-## Get started with PerfTop
+To start the performance analyzer RCA agent on a tarball install, run the following command.
+      
+````bash
+OPENSEARCH_HOME=~/opensearch-2.2.1 OPENSEARCH_JAVA_HOME=~/opensearch-2.2.1/jdk OPENSEARCH_PATH_CONF=~/opensearch-2.2.1/bin ./performance-analyzer-agent-cli
+````
 
-The basic syntax is:
+The below command is used to enable the performance analyzer plugin and performance analyzer RCA agent.
```

Comment:
Recommended write "The command to enable the performance analyzer plugin and performance analyzer RCA agent is:"